com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -35,6 +35,7 @@ internal class Styles
             public static GUIContent hdrText = EditorGUIUtility.TrTextContent("HDR", "Controls the global HDR settings.");
             public static GUIContent msaaText = EditorGUIUtility.TrTextContent("Anti Aliasing (MSAA)", "Controls the global anti aliasing settings.");
             public static GUIContent renderScaleText = EditorGUIUtility.TrTextContent("Render Scale", "Scales the camera render target allowing the game to render at a resolution different than native resolution. UI is always rendered at native resolution. When VR is enabled, this is overridden by XRSettings.");
+            public static GUIContent shaderQualityText = EditorGUIUtility.TrTextContent("Shader Quality", "Controls the URP and Shader Graph shaders quality.");
```

Comment:
It also controls custom user shaders, even if they don't define the macros we infer defaults.

Reply:
Should we use defaults or platform-specific?

Reply:
My suggestion:
Select the Shader Quality tier.